File: Grace/Grace.c
/* Let me present you Grace */
#include <stdio.h>
#define X 1
#define Y 2
#define Z int main(){ FILE *fp = fopen("Grace_kid.c","w");char *s = "/* Let me present you Grace */%c#include <stdio.h>%c#define X 1%c#define Y 2%c#define Z int main(){ FILE *fp = fopen(%cGrace_kid.c%c,%cw%c);char *s = %c%s%c; fprintf(fp, s, 10, 10, 10, 10, 34, 34, 34, 34, 34, s, 34, 10); fclose(fp);}%cZ"; fprintf(fp, s, 10, 10, 10, 10, 34, 34, 34, 34, 34, s, 34, 10); fclose(fp);}
Z

File: Javascript/Grace.js
/*
	Let me present you Grace
*/
const fs = require('fs')

const X = 1
const Y = 2
const Z = main = () => {
	try {
		fs.writeFileSync('Grace_kid.js', `/*\n\tLet me present you Grace\n*/\nconst fs = require('fs')\n\nconst X = 1\nconst Y = 2\nconst Z = main = ${main}\n\nZ()\n`)
	} catch (err) {
		
	}
}

Z()
